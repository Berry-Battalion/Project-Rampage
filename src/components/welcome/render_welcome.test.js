import React from 'react';
import Welcome from './welcome';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router';

describe('<Welcome />', () => {
    test('renders correctly', () => {
        const tree = renderer.create(
            <StaticRouter context={{}}>
                <Welcome />
            </StaticRouter>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
})