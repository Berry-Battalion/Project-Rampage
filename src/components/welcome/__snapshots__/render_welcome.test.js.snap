// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome /> renders correctly 1`] = `
<div
  className="sc-bdVaJa joQndy"
>
  <div>
    Welcome!
  </div>
  <a
    href="/signin"
    onClick={[Function]}
  >
    Sign In
  </a>
</div>
`;
